src/engraving/libmscore/bracket.cpp

```diff
@@ -389,9 +390,6 @@ void Bracket::endEditDrag(EditData&)
     qreal ey = system()->staff(staffIdx2)->y() + score()->staff(staffIdx2)->height();
     h2 = (ey - sy) * .5;
     bracketItem()->undoChangeProperty(Pid::BRACKET_SPAN, staffIdx2 - staffIdx1 + 1);
-    // brackets do not survive layout
-    // make sure layout is not called:
-    score()->cmdState()._setUpdateMode(UpdateMode::Update);
```

Comment:
This is no longer needed, and in fact causes a problem that the barlines are not updated properly